src/screens/Dashboard.js

```diff
@@ -54,9 +54,11 @@ class Dashboard extends React.Component {
     const blocks = this.props.data.length > 0 ? this.props.data[this.props.data.length - 1].blocks : '';
     const transactions = this.props.data.length > 0 ? this.props.data[this.props.data.length - 1].transactions : '';
     const peers = this.props.data.length > 0 ? this.props.data[this.props.data.length - 1].peers : '';
+    const height = this.props.data.length > 0 ? this.props.data[this.props.data.length - 1].best_block_height : '';
     return (
       <div className="content-wrapper">
         <p><strong>Blocks: </strong>{blocks}</p>
+        <p><strong>Height of the best chain: </strong>{height}</p>
```

Comment:
Maybe we should rename it just to height. Or Latest block height. What do you think?

Reply:
I just feel it might be confusing to some people to see "Blocks" and "Height" only and they are really different numbers, that's why I used "Height of the best chain"

I could use "Latest block height" but still the last block could be outside the best chain, so this name would be wrong.

What do you think?

Reply:
I think they say "Latest block" thinking always in the best blockchain. But we can explicitly say "Height of the best chain". It seems good as well. Go ahead!